pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go

```diff
@@ -61,6 +66,11 @@ func (c *connectivityCheckController) WithPodNetworkConnectivityCheckFn(podNetwo
 	return c
 }
 
+type unsupportedConfigOverrides struct {
```

Comment:
I would prefer this to be under a struct like this `.spec.unsupportedConfigOverrides.operator.{disabledControllers,optionalControllers}`.

Reply:
Done.